tuna/driver_base.py

```diff
@@ -66,6 +73,18 @@ def construct_driver(self, line):
 
     return True
 
+  def construct_driver_from_db(self, db_obj):
+    """Takes a <>_config row and returns a driver cmd"""
+    LOGGER.info('Processing db_row: %s', db_obj.to_dict())
+    #common tensor among convolution and batch norm
+    self.decompose_input_t(db_obj)
+    if isinstance(db_obj, ConvolutionConfig):
+      self.parse_conv_row(db_obj)
+    elif isinstance(db_obj, BNConfig):
+      self.parse_bn_row(db_obj)
```

Comment:
Perhaps it would be better to override a parse_row function in the driver child so that we don't leak the config types up into the generic class.